flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64", "aarch64"]
```

Comment:
No need to define that, these are the only supported architectures already.

Reply:
Ok, removed it